II.pass-tests/08-get-average/get-average.js

```diff
@@ -1,3 +1,8 @@
 // the input is an array of numbers and strings
 // return the average of all the numbers
-// be sure to exclude the strings
\ No newline at end of file
+// be sure to exclude the strings
+exports.average = elements => {
+    const numbers = elements.filter(element => typeof element === "number")
+    return numbers.reduce((a, b) => a + b) / numbers.length;
```

Comment:
Like the exercise above, this function might have a problem if you pass an empty array

Reply:
```exports.average = (elements = []) => {
    let len = 1;
    if (elements.length == 0) {
        return 0;
    }
    return elements.slice()
    .filter(element => typeof element === "number")
    .reduce((a, b, cindex) => {
        len = cindex + 1;
        return a + b}, 0) / len;
}```
```test('Average', function () {
    var numbers = [4, '-', 8, 11, 'hello', '57', 0, 2];
    var expected = 5;

    var output = average(numbers);
    expect(average()).toEqual(0);//default
    expect(average([])).toEqual(0);//empty
    expect(output).toEqual(expected);
});```

Reply:
I think passing a test from you is hard :simple_smile:!